docs/content/guides/2.alokai-essentials/1.alokai-next-js/6.product-page.md

```diff
@@ -0,0 +1,358 @@
+  content: [
+    './app/**/*.{js,ts,jsx,tsx}',
+    './components/**/*.{js,ts,jsx,tsx}',
+    '../../node_modules/@storefront-ui/react/**/*.{js,mjs}',
+  ],
+  theme: {
+    extend: {},
+  },
+  plugins: [],
+};
+```
+
+This will configure Tailwind CSS to work with Next.js App Router file structure and Storefront UI.
+
+### Add Tailwind CSS to the Project
+
+In the `app/globals.css` file, add the following code:
+
+```css
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+```
+
+### First Storefront UI Component
+
+Now, let's use the first Storefront UI component in the `app/page.tsx` file. Add the following code to the file:
+
+```diff
+import { getSdk } from "../sdk/sdk.config"
++ import { SfButton } from "@storefront-ui/react"
+
+const sdk = getSdk();
+
+export default async function Page() {
+  const { products } = await sdk.sapcc.searchProduct({});
+
+  console.log(products?.map((product) => product.name));
+
+- return <div>Page</div>
++ return (
++   <div>
++     <SfButton>Click me</SfButton>
++   </div>
++ )
+}
+```
+
+This will import the `SfButton` component from Storefront UI. Here's how the result should look like:
+
+![Storefront UI Button](./images/sfui-1.webp)
+
+Great! Now, let's build the Product Details page with Storefront UI.
+
+## Building Product Details Page
+
+Storefront UI is a very versatile library that provides a lot of components and blocks to build a modern eCommerce application. In this guide, we will build a Product Details page using Storefront UI components and blocks.
+
+First, let's visit Storefront UI documentation to see what components and blocks are available. You can find the [SFUI Documentation here](https://docs.storefrontui.io/v2/react/components.html).
+
+Storefront UI provides two type of components: Base Components and Blocks. Base Components are the basic building blocks of the UI and Blocks are the pre-built components that can be used to build a page. Blocks are made up of Base Components. Blocks are very useful to build a page quickly and easily without writing a lot of code. This is exactly what we need to build a Product Details page.
+
+### Product Details Page Blocks
+
+To build the Product Details we will use the following Storefront UI Blocks:
+
+- [Product Details](https://docs.storefrontui.io/v2/react/blocks/ProductCard.html#details)
+- [Product Gallery with Vertical Thumbnails](https://docs.storefrontui.io/v2/react/blocks/Gallery.html#product-gallery-with-vertical-thumbnails)
+- [Product Slider](https://docs.storefrontui.io/v2/react/blocks/ProductSlider.html)
+
+Let's start building the page!
+
+### Product Details Page
+
+Storefront UI is not like any other UI library. Blocks are too complex to be used directly imported from the library. It would be very difficult and time consuming to customise them. So, instead, Storefront UI team created Blocks to be copied and pasted into the project and then customised. This is a very good approach to build a page quickly and easily.
```

Comment:
```suggestion
Storefront UI is not like most traditional UI libraries. Blocks are too complex to be used directly imported from the library. It would be very difficult and time consuming to customise them. So, instead, Storefront UI team created Blocks to be copied and pasted into the project and then customised. Since all of the source code is available directly to you, including any styles - you have full control to change the styling.
```